### PR TITLE
Improved Checkbox field filter interface

### PR DESCRIPTION
**Pull request by Vultraz**
Before:
image

After:
image

### CODE REVIEW

File: packages/fields/src/Implementation.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ class Field {
     return {};
   }
 
-  /*
+  /**
```
Comment (Vultraz) [inline packages/fields/src/Implementation.js:86]: This just makes this a proper JSDoc comment.

File: packages/fields/src/types/Checkbox/views/Filter.js [hunk 2]
```diff
@@ -1,18 +1,15 @@
 import React from 'react';
+import { RadioGroup, Radio } from '@arch-ui/filters';
 
 const CheckboxFilterView = ({ onChange, filter, innerRef, value }) => {
-  const handleChange = ({ target: { value } }) => {
```
Comment (jossmac) [inline packages/fields/src/types/Checkbox/views/Filter.js:4]: Why was this removed?
